### PR TITLE
Refactoring the edit fields

### PR DESCRIPTION
**Pull request by tomondre**

### CODE REVIEW

File: backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java [hunk 1]
```diff
@@ -30,13 +30,14 @@ public Optional<Account> editAccount(int id, Account account)
...
+            edited.setPassword(account.getPassword());
             edited.setEmail(account.getEmail());
             edited.setName(account.getName());
             edited.setCountry(account.getCountry());
             edited.setProfilePictureUrl(account.getProfilePictureUrl());
-            edited.setPassword(account.getPassword());
-            edited.setEnabled(account.isEnabled());
-            edited.setFavourites(account.getFavourites());
+            edited.setDateOfBirth(account.getDateOfBirth());
+            edited.setGender(account.getGender());
```
Comment (KhaledHammoun) [inline backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java:40]: Dont be too open minded Tomas :D We shouldnt allow gender change :D 
And I think you cannot really change you birthdate

**Review by KhaledHammoun — APPROVED** [review]

File: backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java (hunk 1 above)
Comment (tomondre) [reply backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java:40]: True, I did not realise the meaning of fields, I just saw fields of an object 😂😂😂

Comment (tomondre) [reply backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java:40]: I mean if you can change your gender, you can also change your birthday

Comment (tomondre) [reply backend/src/main/java/com/sep6/backend/repository/AccountsRepositoryImpl.java:40]: We may have customers that complain cuz they cannot change the gender 😆😆

**Pull request merged**